framework/update/helper/swap.cpp

```
@@ -0,0 +1,234 @@
+        if (!platform::waitForProcessExit(args.waitPid, /*timeoutMs*/ 60000)) {
+            logLine("error: host process is still running");
+            return 1;
+        }
+    }
+
+    const fs::path src(args.src);
+    const fs::path dst(args.dst);
+    const fs::path staging = fs::path(dst).concat(".staging");
+    const fs::path backup = fs::path(dst).concat(".bak");
+
+    std::error_code ec;
```

Clang (14.0.6)

[warning] variable name 'ec' is too short, expected at least 3 characters

(readability-identifier-length)